tests/test_openjdk.py

```diff
@@ -61,42 +63,57 @@ def test_jdk_version(container, java_version):
     )
 
 
+@dataclass
+class TestExtendedParams:
```

Comment:
Could you please document the class parameters? At the moment one has to look into the code to find out what format is expected with e.g. `environment`